tags/all/families.csv

```diff
@@ -1953,6 +1953,12 @@ DotGothic16,/Expressive/Vintage,75
 DotGothic16,/Sans/Glyphic,80
 DotGothic16,/Sans/Grotesque,30
 DotGothic16,/Theme/Techno,100
+Doto,/Expressive/Artistic,50

```

Comment:
The only category I would remove is `Artistic` as it was defined as "Flourished Calligraphic & art movements"
Inspecting it in the tagging tool this is indeed what is being listed under it.